genesyscloud/resource_genesyscloud_flow.go

```diff
@@ -58,7 +58,7 @@ func FlowExporter() *resourceExporter.ResourceExporter {
 
 func ResourceFlow() *schema.Resource {
 	return &schema.Resource{
-		Description: `Genesys Cloud Flow`,
+		Description: `Genesys Cloud Architect Flow. Note: Changing any field in this resource will cause the current flow to be dropped and recreated with a new ID.`,
```

Comment:
Is the old flow deleted? If not is that worth mentioning?

Reply:
@dginty4 Yeah it is deleted. That's what "dropped" is supposed to mean in this case

Reply:
Ah ok, sorry I thought it was said in standup it wasn't deleted, my bad

Reply:
Oh yeah that bug was fixed in another PR that's been merged already